src/components/InterestForm/index.tsx

```diff
@@ -0,0 +1,99 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import { UUID } from 'crypto';
+import { insertInterest } from '../../api/supabase/queries/interest';
+import { Interest, CaseListing } from '../../types/schema';
+import { Button } from '../Button';
+import {
+  FormContainer,
+  FormTextArea,
+  FormInput,
+  RadioGroup,
+  RadioLabel,
+  RadioInput,
+  FormFooter,
+  FormWarning,
+} from './styles';
+
+const radioOptions = ['Attorney', 'Interpreter', 'Both Attorney & Interpreter'];
```

Comment:
can we rename 'Both Attorney & Interpreter' to 'Either Attorney Or Interpreter'? David just made that change today!